config/webpacker.yml

```diff
@@ -35,10 +35,10 @@ development:
   # Reference: https://webpack.js.org/configuration/dev-server/
   dev_server:
     https: false
-    host: localhost
+    host: sh
```

Comment:
Should we have localhost be default on the repo? and have instructions to change it if people want to use whales?